integration/fieldmask/fieldmask-test.ts

```diff
@@ -0,0 +1,35 @@
+import { Reader } from 'protobufjs';
+import { FieldMaskMessage } from './fieldmask';
+
+import { FieldMaskMessage as PbFieldMaskMessage } from './pbjs';
```

Comment:
Fwiw we use the `pbjs` types primarily for sanity checking ts-proto core `encode` / `decode` logic. I feel comfortable enough with that code coverage that I'd be fine skipping pbjs types for these `integration/fieldmask` tests, and just use the ts-proto provided encode/decode.

(Although I guess if we were changing the object representation, e.g. `FieldMessageMesage.fieldMask` as a `string[]`, then yeah we'd probably want to sanity check our encode/decode against a 3rd party.)

(It looks like the CI build failed on a pbjs error as well, so the easiest way of fixing that is to probably just skip it all together. If you want to keep pbjs, I think look at the `integration/pbjs.sh` file; for now we hand-maintain that given that most integration tests don't use pbjs types.)

Reply:
I see! I minimized the test to only cover the JSON encoding/decoding.